package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Parasoft Findings enables you to visualize test and static analysis results generated by Parasoft products in Azure DevOps",
   "main": "index.js",
   "scripts": {
+    "download": "cd ./scripts && node downloadReportGenerator.js && cd ..",
```

Comment:
Change to one line:
`node ./scripts/downloadReportGenerator.js`
And change task name to: downloadReportGenerator